From b2f04a659eb9421dd1ee3a1f29516966d22163ee Mon Sep 17 00:00:00 2001
From: Shivani Agarwal <shivania2@vmware.com>
Date: Mon, 10 Jun 2024 09:50:47 +0000
Subject: [PATCH 31/31] ssh denial fix

AVC avc:  denied  { setattr } for  pid=891 comm="sshd" name="root" dev="sda3" ino=398438 scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=system_u:object_r:var_log_t:s0 tclass=file permissive=0

AVC avc:  denied  { create } for  pid=916 comm="sshd" name="root" scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=system_u:object_r:var_log_t:s0 tclass=file permissive=0

Signed-off-by: Shivani Agarwal <shivani.agarwal@broadcom.com>
---
 policy/modules/services/ssh.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/policy/modules/services/ssh.te b/policy/modules/services/ssh.te
index cd034aa..efbb94b 100644
--- a/policy/modules/services/ssh.te
+++ b/policy/modules/services/ssh.te
@@ -744,3 +744,6 @@ optional_policy(`
 	xserver_use_xdm_fds(ssh_agent_type)
 	xserver_rw_xdm_pipes(ssh_agent_type)
 ')
+
+allow sshd_t var_log_t:file { create open read write };
+allow sshd_t var_log_t:file { setattr };
-- 
2.39.4

